packages/hint/docs/contributor-guide/getting-started/events.md

```diff
@@ -15,6 +15,7 @@ their signature, and the `interface` they implement.
 * [`traverse::end`](#traverseend)
 * [`traverse::start`](#traversestart)
 * [`traverse::up`](#traverseup)
+* [`can-evaluate`](#canevaluate)
```

Comment:
Maybe the event should be `can-evaluate::script` or something similar?

Reply:
We can only evaluate JS at the moment but I think it's a good idea in case we add something else in the future.